### PR TITLE
[Draft Pr] Russian translation update.

### PR DESCRIPTION
**Pull request by Kowkodivka**
📦

If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [X] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [X] I have ensured that my code compiles, if applicable.
- [X] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

File: core/assets/bundles/bundle_ru.properties [hunk 1]
```diff
@@ -1002,8 +1002,8 @@ setting.difficulty.hard = Сложная
 setting.difficulty.insane = Безумная
 setting.difficulty.name = Сложность:
 setting.screenshake.name = Тряска экрана
-setting.bloomintensity.name = Bloom Intensity
-setting.bloomblur.name = Bloom Blur
+setting.bloomintensity.name = Интенсивность цвета
```
Comment (antsif-a) [inline core/assets/bundles/bundle_ru.properties:1005]: Разве "bloom" - это не свечение?

```suggestion
setting.bloomintensity.name = Интенсивность свечения
```

File: core/assets/bundles/bundle_ru.properties [hunk 2]
```diff
@@ -1002,8 +1002,8 @@ setting.difficulty.hard = Сложная
...
+setting.bloomblur.name = Размытие увета
```
Comment (antsif-a) [inline core/assets/bundles/bundle_ru.properties:1006]: Очепятка. :eyes:

File: core/assets/bundles/bundle_ru.properties [hunk 3]
```diff
@@ -1139,8 +1139,8 @@ rules.waves = Волны
 rules.attack = Режим атаки
 rules.rtsai = ИИ в реальном времени
 rules.rtsminsquadsize = Минимальный размер отряда
-rules.rtsmaxsquadsize = Max Squad Size
-rules.rtsminattackweight = Min Attack Weight
+rules.rtsmaxsquadsize = Максимальный размер отряда
+rules.rtsminattackweight = Минимальный вес атаки
```
Comment (antsif-a) [inline core/assets/bundles/bundle_ru.properties:1143]: Вес атаки? Не уверен, что игрок поймет, к чему это.

File: core/assets/bundles/bundle_ru.properties [hunk 4]
```diff
@@ -1515,9 +1515,9 @@ block.payload-loader.name = Грузовой загрузчик
 block.payload-loader.description = Загружает жидкости и ресурсы в груз
 block.payload-unloader.name = Грузовой разгрузчик
 block.payload-unloader.description = Выгружает жидкости и ресурсы из груза
-block.heat-source.name = Heat Source
-block.heat-source.description = A 1x1 block that gives virtualy infinite heat.
-block.empty.name = Пусто
+block.heat-source.name = Источник тепла
+block.heat-source.description = Блок 1x1, который дает бесконечное тепло.
```
Comment (antsif-a) [inline core/assets/bundles/bundle_ru.properties:1519]: В оригинале есть "virtually", тут его нет.
```suggestion
block.heat-source.description = Блок 1x1, который дает практически бесконечное тепло.
```

File: core/assets/bundles/bundle_ru.properties [hunk 5]
```diff
@@ -1620,52 +1620,52 @@ block.beam-node.name = Лучевой узел
...
 block.lustre.name = Lustre
-block.scathe.name = Scathe
-block.fabricator.name = Fabricator
-block.tank-refabricator.name = Tank Refabricator
-block.mech-refabricator.name = Mech Refabricator
-block.ship-refabricator.name = Ship Refabricator
-block.tank-assembler.name = Tank Assembler
-block.ship-assembler.name = Ship Assembler
-block.mech-assembler.name = Mech Assembler
+block.scathe.name = Скат
```
Comment (antsif-a) [inline core/assets/bundles/bundle_ru.properties:1642]: :skat: :pray:

File: core/assets/bundles/bundle_ru.properties [hunk 6]
```diff
@@ -591,25 +591,25 @@ research.multiplayer = Только хост может исследовать 
...
+objective.destroyunits.name = Уничтожить боевых единиц
+objective.timer.name = Таймер
+objective.destroyblock.name = Уничтожить блок
+objective.destroyblocks.name = Уничтожить блоки
+objective.destroycore.name = Уничтожить ядро
+objective.commandmode.name = Режим командования
+objective.flag.name = Флаг
+marker.shapetext.name = Текст с меткой
+marker.minimap.name = Мини-карта
+marker.shape.name = Метка
```
[the remark this answers is not in the document]
Comment (Kowkodivka) [reply core/assets/bundles/bundle_ru.properties:609]: На самой карте ставится метка, которой можно менять форму, поэтому, думаю, так будет лучше передавать смысл

File: core/assets/bundles/bundle_ru.properties (hunk 1 above)
Comment (Kowkodivka) [reply core/assets/bundles/bundle_ru.properties:1005]: Сам уже запутался хд

File: core/assets/bundles/bundle_ru.properties (hunk 5 above)
Comment (Kowkodivka) [reply core/assets/bundles/bundle_ru.properties:1642]: :skat: 🛐

**Comment by Anuken** [comment]
When will this be merge-able?

**Comment by Kowkodivka** [comment]
> When will this be merge-able?

Sorry, I don't know exactly. I'll try to finish in 1-2 days. ~I'm lazy and have skill issue with github.~

**Comment by Anuken** [comment]
Some related PRs:

- #6894
- #5391
- #7156
- #7329
- #6888
- #7451

Thoughts on which ones should be merged or not? This needs to be cleaned up.

**Review by RusLeonid — COMMENTED** [review]
This PR is unfinished - these are the issues:
- some of the descriptions aren't exactly correct
- turret names are inconsistent and are mostly translated literally, which isn't ideal
- unit names are mostly translated literally and don't have the same double meaning as in English
- some translations just aren't correct (e.g. vent condenser and flow ducts)
- some translations aren't finished at all (e.g. Uncategorized instructions)

Everything except the aforementioned is well translated.

File: core/assets/bundles/bundle_ru.properties [hunk 7]
```diff
@@ -313,11 +313,11 @@ open = Открыть
...
 command = Командовать
-command.mine = Mine
-command.repair = Repair
-command.rebuild = Rebuild
-command.assist = Assist Player
-command.move = Move
+command.mine = Копать
+command.repair = Ремонтировать
+command.rebuild = Восстанавливать
+command.assist = Помощь игроку
```
[the remark this answers is not in the document]
Comment (xamgore) [reply core/assets/bundles/bundle_ru.properties]: Я бы ещё рассмотрел "Помочь игроку" и "Восстановить".

**Comment by xamgore** [comment]
@Kowkodivka why have you decided to drop the work? :(

**Comment by Leo-MathGuy** [comment]
> This PR is unfinished - these are the issues:
> 
> - some of the descriptions aren't exactly correct
> 
> - turret names are inconsistent and are mostly translated literally, which isn't ideal
> 
> - unit names are mostly translated literally and don't have the same double meaning as in English
> 
> - some translations just aren't correct (e.g. vent condenser and flow ducts)
> 
> - some translations aren't finished at all (e.g. Uncategorized instructions)
> 
> 
> 
> Everything except the aforementioned is well translated.

I can do it if this is still needed, I'm fluent (also my name is also Leonid)

**Comment by Kowkodivka** [comment]
PR давно закрыт, все что оставалось перевести уже перевели.

**Pull request closed without merging**